### ISSUE TITLE
Build on Arduino Leonardo: 'OCR2B' was not declared in this scope

### ISSUE THREAD

**Issue by lumarseg** [trackuino/trackuino#1]
There is a compilation error when the Arduino Leonardo is selected as Board Source:

In file included from afsk.cpp:32:0:
afsk_avr.h: In function 'void afsk_output_sample(uint8_t)':
afsk_avr.h:40: error: 'OCR2B' was not declared in this scope
 #  define OCR2 OCR2B
                ^
afsk_avr.h:53:3: note: in expansion of macro 'OCR2'
   OCR2 = s;
   ^
'OCR2B' was not declared in this scope

**Comment by trackuino** [comment]
As expected, since the atmega32u4 doesn't have a timer 2. That particular code should be ported to use either timer 3 or timer 4.

**Comment by lumarseg** [comment]
So, if I change the setting timer 2 to timer 3 or 4, I need to modify the PCB Gerber files, right?

**Comment by lumarseg** [comment]
We're building a Trackuino over a Arduino Expansion Shield for Raspberry Pi B+ (http://www.dfrobot.com/index.php?route=product/product&product_id=1211&search=Arduino+Expansion+Shield+for+Raspberry+Pi+B%2B&description=true#.VctUNq12Hw0). 
This project is for a charitable cause. It will be launch in August 28th, in Campus Party Costa Rica. But unfortunally we have some troubles to test the prototype. 
The Arduino Expansion Shield is attache to a Raspberry Pi. The Raspberry Pi will take pictures above the ground.
The Radio Club in Costa Rica is enabling 3 digipeters in the highest mountains of the country. Yesterday we had successful results with one Digipeter and the APRS Server to post data in APRS.FI
Mr. Martin, we appreciate so much any cooperation you can give us in this way.

**Comment by trackuino** [comment]
> So, if I change the setting timer 2 to timer 3 or 4, I need to modify the PCB Gerber files, right?

Right, you would have to change the schematic and use some other pin for audio. Pin 3 in the leonardo can do PWM, but it's mapped to Timer 0, which is used by the arduino libraries to keep track of time (delay, millis) so you can't reprogram Ttimer 0 without breaking those.

Also timer2 in the 328p is 8-bit, and timer3/4 in the leonardo are 16/10-bit IIRC. They should be easy to configure to run in 8-bit mode, though.

But, given your timeline, I would suggest you use a Uno and keep the tracker/Pi separate, that way you can use the shield + code unmodified.

**Comment by lumarseg** [comment]
We'll use then an Arduino Uno. In two weeks we'll launch the Balloon. Yestarday our Trackuino Shield didn't work. So in this moment we are in trouble. Our time is -6:00 GMT.
